packages/Icon/src/DragHandle.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+
+const SvgDragHandle = ({ title, ...props }) => (
+  <svg
+    width="1em"
+    height="1em"
+    viewBox="0 0 24 24"
+    css={`
+      color: ${props.color};
+      width: ${props.size};
+      height: ${props.size};
```

Comment:
I dont see any defaults, and you arent passing in these props... are they just omitted if null?

Reply:
Yeah, this is how the `svgr` script generates the component.  It would be nicer if it generated `propTypes` too... but I didn't want to get into AST Templates if I could avoid it (with a simple little eslint exception).